### PR TITLE
fix(css): correct values in the example

### LINKED ISSUE #29503: Invalid oklab() values?
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/oklab

### What specific section or headline is this issue about?

Adjusting opacity with oklab()

### What information was incorrect, unhelpful, or incomplete?

The `a` and `b` coordinates in the CSS colors `oklab(50% 130 20)` and `oklab(50% 130 20 / 0.4)`

### What did you expect to see?

Something like `oklab(50% 0.13 0.2)` or `oklab(50% 130% 20%)`.

I don't know why it would be over 100%, but that would be closer to reasonable than `130` without a percent sign.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

I tried changing the numbers up and down in the playground. It seems that my browser makes different guesses about how to interpret the numbers, when I add the percent sign to `130` or `20`, or both, or neither. Now I'm not sure what kinds of values are actually valid, or if they only work due to some compatibility quirk.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/color_value/oklab`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/oklab
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/color_value/oklab/index.md
* Last commit: https://github.com/mdn/content/commit/8cb896db41ef17577941067991a2a805d7f661e4
* Document last modified: 2023-07-17T08:05:41.000Z

</details>

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fixes https://github.com/mdn/content/issues/29503

`a` and `b` axis values should be in range `-0.4` to `0.4`.

### CODE REVIEW

File: files/en-us/web/css/color_value/oklab/index.md [hunk 1]
```diff
@@ -162,3 +162,4 @@ div {
 - The [`<color>` data type](/en-US/docs/Web/CSS/color_value) for a list of all color notations
 - {{cssxref("color_value/oklch",'oklch()')}}: Another functional notation using the same color space as `oklab()` but in a polar coordinate system
 - [A perceptual color space for image processing](https://bottosson.github.io/posts/oklab/)
+- [OKLAB color wheel](https://observablehq.com/@shan/oklab-color-wheel)
```
Comment (dipikabh) [inline files/en-us/web/css/color_value/oklab/index.md:165]:
```suggestion
- [A perceptual color space for image processing](https://bottosson.github.io/posts/oklab/) on bottosson.github.io (2023)
- [OKLAB color wheel](https://observablehq.com/@shan/oklab-color-wheel) on observablehq.com
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for these fixes, @OnkarRuikar. 

> `a` and `b` axis values should be in range `-0.4` to `0.4`.

Nothing wrong with the updated values but technically, `a` and `b` can be specified as percentages too, correct?

**Comment by OnkarRuikar** [comment]
> Thanks for these fixes, @OnkarRuikar.
> 
> > `a` and `b` axis values should be in range `-0.4` to `0.4`.
> 
> Nothing wrong with the updated values but technically, `a` and `b` can be specified as percentages too, correct?

Yes, 100% is 0.4.

**Comment by dipikabh** [comment]
> It's hard to understand these new color spaces with rgb mindset. We really need to have 3d illustrations of theses color spaces on these pages. There is no image of oklab space on the Internet.

It is difficult, I agree. And the best we can do at the moment is to add links to closest relevant articles (we've already linked to the blog of the oklab creator). I looked for others and came across this: [An interactive review of Oklab](https://raphlinus.github.io/color/2021/01/18/oklab-critique.html) - it does not provide 3D illustrations, but might be useful to compare and understand differences between Oklab and other color spaces. WDYT about adding this resource as well to "See also"?

**Review by dipikabh — APPROVED** [review]
Thank you! 👍

**Pull request merged**